docs/patterns.rst

```diff
@@ -0,0 +1,71 @@
+Patterns of Use
+===============
+
+The patterns contained in this module support our use cases for services hosted at Thunderbird. At an extremely high
+level, this module allows us to standardize certain aspects of the Pulumi resources we're building by using a custom
+``ComponentResource`` called a :py:class:`tb_pulumi.ThunderbirdComponentResource`. When building your Pulumi project
+using this module, you should organize these resources into a :py:class:`tb_pulumi.ThunderbirdPulumiProject`.
+
+The following patterns are available:
+
+* :py:class:`tb_pulumi.cloudfront.CloudFrontS3Service`: Store static content in an S3 bucket and serve it up over the
+  CloudFront Content Delivery Network.
```

Comment:
Should this be a newline?

Reply:
Yes, it's 120-character line widths. It ends up looking weirder here because none of the other lines need to wrap.